src/pages/test-report/components/end-test-modal/end-test-modal.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { IonicPageModule } from 'ionic-angular';
+import { EndTestModal } from './end-test-modal';
+
+@NgModule({
```

Comment:
The declarations and imports can be moved to `test-report-components.module.ts`

Reply:
The modal is not really a component, it's a page, so I think it needs to be like this?

Reply:
From a short bit of investigation while I was doing modals, it seemed that `@IonicPage` was required, could be wrong though.